dhall-json/tasty/data/error/unhandledKeysSchema.dhall

```diff
@@ -0,0 +1 @@
+List { a: Natural }
```

Comment:
Another missing newline. You can probably configure your editor to add these by default. Or maybe you have a funny git config!?

Reply:
haha! I've just configured my editor to add them automatically.